From 2370ccf579bd0fb4484c343389b8121c6b7f9bb8 Mon Sep 17 00:00:00 2001
From: Keith Smiley <keithbsmiley@gmail.com>
Date: Sat, 20 Sep 2025 20:02:06 -0700
Subject: [PATCH] PR #1939: Add missing rules_cc loads

Imported from GitHub PR https://github.com/abseil/abseil-cpp/pull/1939

This is required for use with bazel past this commit: https://github.com/bazelbuild/bazel/commit/71ca0ed111ff3d842a0d23bc3a46bd2e6745491d

Most files do this already these were just missing

Merge f7d58947afdb0549c9e5c287a5e5851a7d72e1e0 into 620d600442769b7ec4b61bd87667899908eec4ef

Merging this change closes #1939

COPYBARA_INTEGRATE_REVIEW=https://github.com/abseil/abseil-cpp/pull/1939 from keith:ks/add-missing-rules_cc-loads f7d58947afdb0549c9e5c287a5e5851a7d72e1e0
PiperOrigin-RevId: 809556509
Change-Id: I494a118b851685a03393485514a3b6fbe3a10597
---
 absl/debugging/BUILD.bazel | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/absl/debugging/BUILD.bazel b/absl/debugging/BUILD.bazel
index ed7cc493e90..7cc053e783e 100644
--- a/absl/debugging/BUILD.bazel
+++ b/absl/debugging/BUILD.bazel
@@ -14,6 +14,9 @@
 # limitations under the License.
 #
 
+load("@rules_cc//cc:cc_binary.bzl", "cc_binary")
+load("@rules_cc//cc:cc_library.bzl", "cc_library")
+load("@rules_cc//cc:cc_test.bzl", "cc_test")
 load(
     "//absl:copts/configure_copts.bzl",
     "ABSL_DEFAULT_COPTS",
